doc/crud.md

```diff
@@ -191,6 +233,26 @@ rev.attachments[att1.name] = att1;
 CDTDocumentRevision *saved = [datastore createDocumentFromRevision:rev
                                                              error:&error];
 ```
+```swift
+// Create a new document
+let rev = CDTDocumentRevision()
+// or get an existing one:
+let rev = try datastore.getDocumentWithId("mydoc")
+
+rev.body = [....]
+
+let att1 = CDTUnsavedFileAttachment(path: "/path/to/image/jpg",
+    name: "cute_cat.jpg",
+    type: "image/jpeg")
+
+// As with the document body, you can replace all the attachments:
+rev.attachments = [att1.name : att1]
+// or just add or update a single one
```

Comment:
Very minor but add a `:` to the end of the comment to match the Objective C example. `// or just add or update a single one:`